etp-backend/src/main/sql/solita/etp/db/sivu.sql

```diff
@@ -21,6 +21,9 @@ where
   id = :id and
   (:paakayttaja or published);
 
+-- name: delete-sivu!
```

Comment:
Täällä voisi varmaan olla myös WHERE-ehto, että sivulla ei saa lapsisivuja (subquery). Sitten servicessä voisi tarkistaa lukumäärän kuinka monta riviä muuttui, ja palauttaa sen api:iin. Muistaakseni tätä samaa ideaa käytetään jossain muuallakin.

Reply:
Ajattelin vain ottaa FKE:n kiinni ja antaa siitä bad request -vastauksen, mutta tämä voisi olla myös ihan suht simppeli reitti hoitaa se asia.

Reply:
Nyt äskettäin pushatussa versiossa olisi `:foreign-key-violation` otettu kiinni ja sen perusteella 400 status.